io.github.mpc_qt.Mpc-Qt.yml

```diff
@@ -0,0 +1,167 @@
+---
+app-id: io.github.mpc_qt.Mpc-Qt
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: mpc-qt
+rename-icon: mpc-qt
+rename-desktop-file: mpc-qt.desktop
+copy-icon: true
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=pulseaudio'
+  - '--device=dri'
+  - '--filesystem=host'
+  - '--talk-name=org.freedesktop.ScreenSaver'
+  - '--system-talk-name=org.freedesktop.UDisks2'
+  - '--own-name=org.mpris.MediaPlayer2.MpcQt'
+  - '--env=LC_NUMERIC=C'
```

Comment:
do you really need numbers to be displayed the wrong in most places in the world?

Reply:
That's required by mpv

Reply:
That can go because we actually set it LC_NUMERIC in main.cpp.